release_bot/configuration.py

```diff
@@ -43,9 +45,29 @@ def __init__(self):
         self.github_app_installation_id = ''
         self.github_app_id = ''
         self.github_app_cert_path = ''
+
         self.clone_url = ''
         self.webhook_handler = False
 
+        # when you need to have different name for pypi release
+        self.pypi_project = ''
+
+
+    def pypi_project_from_setup_cfg(self):
+
+        """
+        Getting the name from the metadata section of setup.cfg
+        :return the pypi_project name or None
+        """
+
+        pypi_config = configparser.ConfigParser()
+        pypi_config.read("setup.cfg")
```

Comment:
If the file does not exist (which can happen easily) pypi_config will be `[]`. And you should also read the values into a variable, so:
```
pypi_config = pypi_config.read("setup.cfg")
```